### PR TITLE
Closes #871: Add links for Github interaction to the wiki page

### LINKED ISSUE #871: Create a "Working with Eclipse Collections on Github" doc
This doc is either a standalone doc or a section in the CONTRIBUTING.md.

Content is already available and shared with @donraab and @nikhilnanivadekar.

Pending a decision on where to house, after which point, a PR can be submitted.

### PR DESCRIPTION
**Pull request by cguntur-bnym**
Signed-off-by: Chandra Guntur <chandra.guntur@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Please add a description of your changes to the [draft release notes](RELEASE_NO
 
 Finally, submit a pull request.
 
+Details on [working with Github for Eclipse Collections](https://github.com/eclipse/eclipse-collections/wiki/Working-with-Github) is located at the Wiki.
```
Comment (sendilkumarn) [inline CONTRIBUTING.md:26]:
```suggestion
Details on [working with GitHub for Eclipse Collections](https://github.com/eclipse/eclipse-collections/wiki/Working-with-Github) is located at the Wiki.
```

File: README.md [hunk 2]
```diff
@@ -89,6 +89,7 @@ We welcome contributions! We accept contributions via pull requests here in GitH
 * StackOverflow: http://stackoverflow.com/questions/tagged/eclipse-collections
 * Mailing lists: https://dev.eclipse.org/mailman/listinfo/collections-dev
 * Forum: https://www.eclipse.org/forums/index.php?t=thread&frm_id=329
+* Working with Github: https://github.com/eclipse/eclipse-collections/wiki/Working-with-Github
```
Comment (sendilkumarn) [inline README.md:92]:
```suggestion
* Working with GitHub: https://github.com/eclipse/eclipse-collections/wiki/Working-with-Github
```

**Comment by donraab** [comment]
@cguntur-bnym please make the changes requested by @sendilkumarn and rebase your PR and I will merge.

**Comment by nikhilnanivadekar** [comment]
@c-guntur can you please update the PR with the suggested changes? It will be good to merge it. Thanks!

**Comment by nikhilnanivadekar** [comment]
@cguntur-bnym any updates?

**Comment by nikhilnanivadekar** [comment]
Addressed via #951

**Pull request closed without merging**